### PR TITLE
Porting to Alpine images

### PR DESCRIPTION
**Pull request by ManoMarks**
Porting to Alpine images, and removing links which aren't necessary.

### CODE REVIEW

File: result-app/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM node:0.10
+FROM mhart/alpine-node
```
Comment (bfirsh) [inline result-app/Dockerfile]: We should use official images in our example app.

Some of the official repos have alpine versions (e.g. [golang](https://hub.docker.com/_/golang/)). Maybe we should add an alpine version for node?

Alternatively, there is a "slim" version of the node image which might be smaller?

File: docker-compose.yml [hunk 2]
```diff
@@ -7,8 +7,6 @@ services:
      - ./voting-app:/app
     ports:
       - "5000:80"
-    links:
```
Comment (bfirsh) [inline docker-compose.yml]: Links do work with new networking, and I kinda like the explicitness of them. It makes it easier to explain to beginners what connects to what, etc.

Also means we don't have to use the "container_name" option used below, I think?

**Comment by bfirsh** [comment]
Looks great apart from a couple of comments - thanks. 😄

File: result-app/Dockerfile (hunk 1 above)
Comment (ManoMarks) [inline result-app/Dockerfile]: I'll test this with a slim node version, there's no official Alpine version and this was the one the Alpine maintainer recommended.

**Comment by ManoMarks** [comment]
fixing the comments @bfirsh had

**Comment by bfirsh** [comment]
Nice, thanks!

**Pull request merged**